content/800-data-platform/200-projects/110-create-new-application-from-template.mdx

```diff
@@ -0,0 +1,105 @@
+        </Admonition>
+    2. From the **GitHub Accounts & Organizations** drop-down menu, select a GitHub account or an organization.
+    3. Select the **Create a repository** option.
+    4. (Optional) In the **Repository Name** text box, enter a name for the repository.
+
+        <Admonition type="info">
+        
+        **Note**<br /><br />
+
+        • The GitHub repository name is pre-filled based on the display name and each space is replaced with a hyphen.<br /><br />
+        • The GitHub repository name cannot duplicate other repository names.<br />
+        </Admonition>
+
+    5. Click **Next**.
+        <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666310180/docs/pdp/create-manage-projects/create-project-01-configure-project_hpijvd.png"
+        alt="Prisma Data Platform - Manage project members - Step 03 - Click Edit for the project member you want to delete"
+        width="750px"
+        />
+3. On the _Select template_ page, select an application template for your first project.
+    1. Select one of the available application templates and preview its Prisma schema file below.
+        * **Blog**. Schema for a blog built with Next.js.
+        * **Music Streaming Service**. Schema for a fictional music streaming service with a REST API.
+        * **Rentals Platform**. Schema for a fictional hotel with a REST API.
+        * **SaaS**. Schema for an identity service with a SaaS with a REST API.
+        * **URL Shortener**. Schema for a URL shortener with a REST API.
+        * **Empty**. An empty schema with no models, to build your own schema.
+    2. (Optional) Deselect the **Include sample data** check box to skip seeding the database with sample data.
+
+        <img
+            src="https://res.cloudinary.com/prismaio/image/upload/v1666310742/docs/pdp/create-manage-projects/create-project-02-deselect-include-sample-data_uo0nzp.png"
+            alt="Prisma Data Platform - Manage project members - Step 03 - Click Edit for the project member you want to delete"
+            width="750px"
+        />
+    3. Click **Next**.
+4. On the _Configure environment_ page, connect your database and configure the Data Proxy location.
+    1. In the **Connection string** text box, paste your database connection string.
+        <Admonition type="alert">
+
+        **Important**<br /><br />
+        If you provide a connection string to a non-empty database (with existing tables and data), the project creation fails.
+        </Admonition>
+    2. Under _Data Proxy_, from the **Location** drop-down menu, select the geographic location for the Data Proxy.
+        <Admonition>
+        
+        **Note**<br /><br />
+        Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+        If you select a different location, when your application sends requests to your database through the Data Proxy, the responses might be slow or might time out due to higher latency.
```

Comment:
Consider adding "then" here. IMO that makes it slightly clearer.

"If you select a different location, then when..."

I had to re-read the start of the sentence, because I wasn't initially sure if this meant "If you select a different location at the time that your application sends requests". Or maybe it does mean this? In which case, maybe a then would after "Data Proxy" would make it clearer?